settings.gradle

```diff
@@ -26,8 +26,14 @@ dependencyResolutionManagement {
                 version('koin-version','3.3.2')
                 version('lottieVersion', '5.2.0')
                 version('ktorVersion', '1.5.0')
+                version('gsonVersion', '2.10.1')
+                version('androidCompose','3.4.1')
+
+                library('koinAndroid','io.insert-koin','koin-android').versionRef('koin-version')
+                //library('koin-compose','io.insert-koin','koin-compose').versionRef('koin-version')
```

Comment:
Remove those libraries  if you don't use them

Reply:
resolved 👍